docs/DeveloperGuide.md

```diff
@@ -295,30 +295,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `iVolunteer` app and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 1: Delete an event**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all events
+2.  iVolunteer shows a list of events
+3.  User requests to delete a specific event in the list
+4.  iVolunteer deletes the person
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 1b. The given command is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 1a1. iVolunteer requests for the correct command.
+    * 1a2. User enters correct command <br>
+    Steps 1a1-1a2 are repeated until the data entered is correct <br>
```

Comment:
Likewise, based on @jordankanghm suggestion:
```suggestion
    * 1b2. User enters correct command <br>
    Steps 1b1-1b2 are repeated until the data entered is correct <br>
```

Reply:
Not sure what you mean. Isn't what I wrote correct?

Reply:
you changed it alr so that's fine haha